packages/blockprotocol/package.json

```diff
@@ -2,6 +2,9 @@
   "name": "blockprotocol",
   "version": "0.0.1",
   "description": "TypeScript typings for https://blockprotocol.org developers",
+  "keywords": [
+    "blockprotocol"
```

Comment:
```suggestion
    "blockprotocol", 
    "blocks"
```

Reply:
Thanks! I added to all three publishable packages.